google/cloud/aiplatform/datasets/dataset.py

```diff
@@ -143,118 +180,91 @@ def create(
                 labels specified inside index file refenced by
                 [import_schema_uri][google.cloud.aiplatform.v1beta1.ImportDataConfig.import_schema_uri],
                 e.g. jsonl file.
-            project: Optional[str]=None,
+            project: Optional[str] = None,
                 Project to upload this model to. Overrides project set in
                 aiplatform.init.
-            location: Optional[str]=None,
+            location: Optional[str] = None,
                 Location to upload this model to. Overrides location set in
                 aiplatform.init.
-            credentials: Optional[auth_credentials.Credentials]=None,
+            credentials: Optional[auth_credentials.Credentials] = None,
                 Custom credentials to use to upload this model. Overrides
                 credentials set in aiplatform.init.
-            sync (bool):
+            request_metadata: Optional[Sequence[Tuple[str, str]]] = ()
+                Strings which should be sent along with the request as metadata.
+            sync: (bool) = True
                 Whether to execute this method synchronously. If False, this method
                 will be executed in concurrent Future and any downstream object will
                 be immediately returned and synced when the Future has completed.
+
         Returns:
             dataset (Dataset):
                 Instantiated representation of the managed dataset resource.
+
+        Raises:
+            ValueError if import_schema_uri is set when creating a tabular dataset.
         """
-        utils.validate_display_name(display_name)
 
-        is_tabular_dataset_metadata = (
-            metadata_schema_uri == schema.dataset.metadata.tabular
-        )
+        utils.validate_display_name(display_name)
 
-        if bool(bq_source) and not is_tabular_dataset_metadata:
-            raise ValueError(
-                "A tabular metadata_schema uri must be used when using a BigQuery source"
-            )
+        # Validate metadata_schema_uri and import_schema_uri for Dataset class
+        cls._validate_import_schema_uri(import_schema_uri)
 
-        api_client = cls._instantiate_client(location=location, credentials=credentials)
+        if metadata_schema_uri == schema.dataset.metadata.tabular:
```

Comment:
We may have to refactor this slightly to allow for forecasting as well. Fine to leave as-is for now and modify for forecasting later.